src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -66,9 +66,12 @@ const VARIANTS = {
     UltimateChickenHorse: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
     AtrioTheDarkWild: [new ModLoaderPackageMapping("BepInEx-BepInExPack", "BepInExPack", PackageLoader.BEPINEX)],
     AncientDungeonVR: [],
-    Brotato: [
-        // TODO: Add godot mod loader support
-    ],
+    Brotato: [],
```

Comment:
Is there no `GodotModding-GodotModLoader` package for Brotato?

Reply:
Brotato has it built in